jogo.js

```diff
@@ -69,58 +98,36 @@ const calcularVencedor = () => {
 // Aqui vai tudo responsável por exibir a aplicação para o usuário, bem como reagir às ações dele
 
 window.onload = function () {
-  // Botões de marcar
-/*  const botoes = [
-    document.getElementById('marcar-0-0'),
-    document.getElementById('marcar-0-1'),
-    document.getElementById('marcar-0-2'),
-    document.getElementById('marcar-1-0'),
-    document.getElementById('marcar-1-1'),
-    document.getElementById('marcar-1-2'),
-    document.getElementById('marcar-2-0'),
-    document.getElementById('marcar-2-1'),
-    document.getElementById('marcar-2-2')
-  ];
-
-  const criarHandlerParaMarcar = (i, j) => function () {
-    this.disabled = true;
+  // Campos para marcação
+  const criarHandlerParaMarcar = (i, j, touch) => function () { 
```

Comment:
O parâmetro `touch` é desnecessário. Como a `function` é usada como event listener, `this` vai fazer referência ao elemento que está ativando o evento. Por exemplo, se você adiciona um event listener a um elemento `xyz`, o valor de `this` é igual a `xyz` na função sendo chamada.